src/Files.App/UserControls/Settings/SettingsBlockControl.xaml

```diff
@@ -8,6 +8,7 @@
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	d:DesignHeight="300"
 	d:DesignWidth="400"
+	Loaded="SettingsBlockControl_Loaded"
```

Comment:
The plan is to switch to the Toolkits control when it becomes available so let's leave the current control as is and revert these changes.

Reply:
It's done.